src/components/dashboard/main-content/course-enrollments/course-cards/CompletedCourseCard.jsx

```diff
@@ -17,15 +17,15 @@ const CompletedCourseCard = props => {
             <img src={CertificateImg} alt="verified certificate preview" />
           </div>
           <div className="d-flex align-items-center">
-            <p className="mb-0 lead">
+            <p className="mb-0 lead font-weight-normal">
```

Comment:
Nit: Seems like this might belong in the other part of the PR, but not a huge deal